src/Components/Transportation/Transportation.test.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "react-native";
+import Transportation from "./Transportation";
+import { shallow } from "enzyme";
+
+describe("Transportation", () => {
```

Comment:
If you want to get a second snapshot, we can pass in some mock transports as a prop to the wrapper in a new it statement and it should run through the generateTransportationElements method to give us a full component (I know that time is of the essence, though ...)